CHANGELOG.md

```diff
@@ -4,8 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.4.0]
```

Comment:
Actually you shouldn't change this manually. We should keep the `Unreleased`

In the `Unreleased` section we add everything that is not in a release yet using:
`### Added`, `### Changed` our `### Updated`.

Later, when any lib maintainer wants to generate a release, it just needs to run `make release` and the changelog will be updated automatically.

Reply:
Later, if you want to make a release, locally in the master branch run

`make release`, `git push origin master` and `git push origin master --tags`

Reply:
Oh, got it! Fixed! Tks